enterprise/migrations/0093_enable_enterprise_catalog_api_endpoints.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
```

Comment:
@bbaker6225 merged in his PR that adds this same migration and `USE_ENTERPRISE_CATALOG` constant. If you rebase this PR's branch on `master`, you'll no longer need this migration in the PR.

Reply:
👍